DB/src/main/java/io/deephaven/db/v2/select/AbstractConditionFilter.java

```diff
@@ -149,6 +149,12 @@ public synchronized void init(TableDefinition tableDefinition) {
             // check if this is a filter that uses a numba vectorized function
             for (Param param : params) {
                 if (param.getValue().getClass() == NumbaCallableWrapper.class) {
+                    /*
+                     * numba vectorized function must be used alone as an expression, and that should be checked in
+                     * the DBLanguageParser
+                     */
+                    assert (params.length == 1);
+
```

Comment:
We typically use `new RuntimeException`.

Reply:
Actually, some flavor of `UncheckedDeephavenException`. Or `Assert` if we're feeling lazy.

Reply:
changed to throw UncheckedDeephavenException